library/lua/argparse.lua

```diff
@@ -80,6 +80,7 @@ end
 -- it will be passed to the handler function.
 -- longOptionAlias is optional.
 -- hasArg defaults to false.
+-- to have an option that has only a long form, pass '' as the shortOptionName.
```

Comment:
I might suggest allowing or supporting `nil` as the sentinel here (with the caveat that `#optionAction` would behave unpredictably, but it looks like you're only accessing `optionAction[1]` and other fields directly)

Reply:
good idea. done.